hera_qm/vis_metrics.py

```diff
@@ -1,35 +1,27 @@
 import numpy as np
-from hera_qm.datacontainer import DataContainer
 
 
-def check_noise_variance(data, wgts, bandwidth, inttime):
+def check_noise_variance(data):
     '''Function to calculate the noise levels of each baseline/pol combination,
     relative to the noise on the autos.
 
     Args:
-        data (dict): dictionary of visibilities with keywords of pol/ant pair
-            in any order.
-        wgts (dict): dictionary of weights with keywords of pol/ant pair in any order
-        bandwidth (float): Channel width, preferably Hz, but only restriction is
-            bandwidth units are recipricol inttime units.
-        inttime (float): Integration time, preferably seconds, but only restriction
-            is inttime units are recipricol bandwidth units.
+        data (UVData): UVData object with data.
```

Comment:
This is a great change, it's much more straightforward to be using a UVData object